index.html

```diff
@@ -0,0 +1,1048 @@
+              href="./blog-details.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Blog
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./contacts.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__assets-block list">
+        <form action="#" method="get" class="assets-block__searching">
+          <label for="asset-searching" class="searching__label hidden"
+            >Find asset</label
+          >
+          <input
+            type="text"
+            class="searching__input input input__small"
+            id="asset-searching"
+            placeholder="Search for Digital Assets"
+          />
+          <button type="submit" class="searching__button hidden">Search</button>
+        </form>
+        <a
+          href="#"
+          class="assets-block__button button button__primary button__square-medium"
+        >
+          <svg class="assets-block__icon icon">
+            <use xlink:href="./images/icons/sprite.svg#bag"></use>
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="container__main">
+      <section class="main">
+        <div class="main__main-information list list__column">
+          <h2 class="main-information__title hidden heading__section">
+            Introduction
+          </h2>
+          <strong class="main-information__highlite-text heading__main-title">
+            Craft Your Success with Our&nbsp;<span class="highlite-text__span"
+              >Digital
+            </span>
+            Assets
+          </strong>
+          <p class="main-information__description text__main">
+            From our inspiring illustrations to dynamic UI/UX kits, elevate your
+            projects with our premium digital design resources.
+          </p>
+          <div class="main-information__main-buttons list">
+            <a
+              href="./catalog.html"
+              class="main-buttons__button button button__primary button__large"
+              >Shop Now</a
+            >
+            <a
+              href="#"
+              class="main-buttons__button button button__navigation button__large"
+              target="_blank"
+            >
+              <svg class="main-buttons__icon icon">
+                <use xlink:href="./images/icons/sprite.svg#play"></use>
+              </svg>
+              Watch About Us
+            </a>
+          </div>
+        </div>
+      </section>
+      <section class="products container__section">
+        <h2 class="hidden">Products</h2>
+        <ul class="products__list list list__elements">
+          <li>
+            <a class="products__link" href="./catalog.html">
+              <article class="products__list-item list list__column">
```

Comment:
The <article> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a self-contained composition in a document. 

I think we can't use it as a self-contained composition, so you need to remove this tag.

Reply:
Fixed 🥇